templates/_base.html.twig

```diff
@@ -2,18 +2,24 @@
 <html lang="en-GB">
   <head>
     <title>{{ subject }}</title>
-    <style type="text/css">
-    p {
-      font-size: 1.2rem;
-    }
-    </style>
   </head>
 <body>
-  <div id="content">{% block content %}{% endblock %}</div>
-  <div id="footer">
-    {% block footer %}
-        The Big Give xx
-    {% endblock %}
+  <div style="margin: 0 auto; max-width:500px;">
```

Comment:
How about a class or ID to keep the markup readable? The point of the CSS inlining plugin is to support this.

Reply:
@NoelLH Just a quick one on this, I tried running this command `composer req twig/cssinliner-extra` following this [guide](https://twig.symfony.com/doc/2.x/filters/inline_css.html) but for some reason composer is complaining I don't have redis extension:

![image](https://user-images.githubusercontent.com/47176130/82347683-d8196d80-99ef-11ea-9588-56d34b7f427a.png)

I've downloaded the windows redis binary but it's still not liking it, any ideas?

Reply:
@tbg-jasonfung you need to install the Redis PHP extension as it says, as well as a Redis client. Or just run this inside the Docker container, which already has both.

> @NoelLH Just a quick one on this, I tried running this command `composer req twig/cssinliner-extra` following this [guide](https://twig.symfony.com/doc/2.x/filters/inline_css.html) but for some reason composer is complaining I don't have redis extension:
> 
> ![image](https://user-images.githubusercontent.com/47176130/82347683-d8196d80-99ef-11ea-9588-56d34b7f427a.png)
> 
> I've downloaded the windows redis binary but it's still not liking it, any ideas?

Reply:
As discussed, this should also be unnecessary. The plugin detailed in the readme is already registered with SwiftMailer which should achieve the same thing without the extra template boilerplate.